jupyter-extension/src/plugins/mount/components/Pipeline/hooks/usePipeline.tsx

```diff
@@ -22,24 +30,43 @@ export type usePipelineResponse = {
 
 export const usePipeline = (
   ppsContext: PpsContext | undefined,
-  saveNotebookMetaData: (metadata: SameMetadata) => void,
+  saveNotebookMetaData: (metadata: PpsMetadata) => void,
 ): usePipelineResponse => {
   const [loading, setLoading] = useState(false);
-  const [pipelineName, setPipelineName] = useState('');
+  const [pipeline, _setPipeline] = useState({name: ''} as Pipeline);
   const [imageName, setImageName] = useState('');
   const [inputSpec, setInputSpec] = useState('');
   const [requirements, setRequirements] = useState('');
   const [errorMessage, setErrorMessage] = useState('');
 
+  const setPipeline = (input: string) => {
```

Comment:
I'd like to test this method individually, but I can't figure out how to do so. Any advice?